docs/design/operand-deployment-lifecycle-manager.md

```diff
@@ -42,262 +42,195 @@ The ODLM will have Three CRDs:
 
 ## OperandRegistry Spec
 
-OperandRegistry defines the operand deployment info:
+OperandRegistry defines the OLM information, like channel and catalog source, for each operator.
 
-* namespace, the namespace of the individual operand
-* channel, the channel(version) of the individual operand
-* packageName, the package name in operator registry of the individual operand
-* sourceName, the name of operator registry
-* sourceNamespace, the namespace of operator registry
+**NOTE:** When the ODLM operator is deployed, it generates a default OperandRegistry instance. You can edit the instance as required.
 
-The OperandRegistry managed operator subscriptions will have a label `"operator.ibm.com/opreq-control": "true"`, we use this label to distinguish if an operator is managed by OperandRegistry or not.
+Following is an example of the OperandRegistry CR:
 
-> **NOTE: These operands will not be installed by default, unless you explicitly set them in OperandRequests**
+**NOTE:** The "name" parameter must be unique for each entry.
 
 ```yaml
-apiVersion: operator.ibm.com/v1alpha1
+apiVersion: odlm.ibm.com/v1alpha1
```

Comment:
odlm ?

Reply:
Thanks, @DanielXLee nice catch. I will fix it and review the document again